gobblin-metrics/src/main/java/gobblin/metrics/kafka/KafkaAvroReporter.java

```diff
@@ -103,4 +119,24 @@ public KafkaAvroReporter build(String brokers, String topic) {
   protected Encoder getEncoder(OutputStream out) {
     return EncoderFactory.get().binaryEncoder(out, null);
   }
+
+  @Override
+  protected void writeSchemaVersioningInformation(DataOutputStream outputStream)
```

Comment:
In addition to doing what the javadoc of the super method says, this method also registers a schema to the registry. It's better to either explain it in the javadoc, or register the schema in another method.